starter_code/script.js

```diff
@@ -3,13 +3,29 @@ var aiPoint = 0;
 
 // This function returns the selection of the computer
 function getAISelection() {
-    //TODO: randomly choose between 'rock', 'paper', or 'scissors'
+    var options = ["rock","scissors","paper"];
+    var random = Math.floor(Math.random() * (0 - 3 + 1)) + 0;
+    return options[random];
 }
 
 // This function picks the winner
 function pickWinner(userValue, aiValue) {
     //TODO: pick the correct winner: user or ai
     //TODO: Add one point for the winner
+    if(userValue == "rock" && aiValue == "paper"){
+        return "winner";
+        userPoint += 1;
```

Comment:
if you return before incrementing the counter, it will **never** be incremented :sob: